bin/rails

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+# This command will automatically be run when you run "rails" with Rails 4 gems installed from the root of your application.
+
+ENGINE_ROOT = File.expand_path('../..', __FILE__)
+ENGINE_PATH = File.expand_path('../../lib/copilot/engine', __FILE__)
```

Comment:
this came straight from the rails generator: rails generate plugin foobar --mountable